package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ui-toolkit",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
See here for details: https://openedx.atlassian.net/wiki/display/FEDX/UI+Toolkit+release+process